eahub/localgroups/models.py

```diff
@@ -15,27 +15,20 @@ class LocalGroupType(enum.Enum):
     COUNTRY = 2
     UNIVERSITY = 3
 
-    labels = {CITY: "City", COUNTRY: "Country", UNIVERSITY: "University"}
+    labels = {CITY: "City", COUNTRY: "National/Regional", UNIVERSITY: "University"}
```

Comment:
Ideally would be great to rename the constant as well

It's definitely a really small thing but I'm slightly concerned with those in the long run. Although I have the impression that in the next ~6 month we might even make the types dynamic (similarly to tags) and that wouldn't be problem anymore.